docs/pages/templates/[slug].tsx

```diff
@@ -0,0 +1,119 @@
+import { GetStaticProps, InferGetStaticPropsType } from 'next';
+import { MDXRemote } from 'next-mdx-remote';
+import { H1 } from '@ag.ds-next/heading';
+import { Flex, Stack } from '@ag.ds-next/box';
+import { ExternalLinkIcon } from '@ag.ds-next/icon';
+import { Body } from '@ag.ds-next/body';
+import { ButtonLink } from '@ag.ds-next/button';
+import { Text } from '@ag.ds-next/text';
+
+import {
+	getTemplate,
+	// getTemplateBreadcrumbs,
+	getTemplateList,
+	getTemplateSlugs,
+	Template,
+} from '../../lib/mdxUtils';
+
+import { mdxComponents } from '../../components/utils';
+import { AppLayout } from '../../components/AppLayout';
+import { DocumentTitle } from '../../components/DocumentTitle';
+import { PageLayout } from '../../components/PageLayout';
+
+export default function Templates({
+	template,
+	templateLinks,
+}: InferGetStaticPropsType<typeof getStaticProps>) {
+	return (
+		<>
+			<DocumentTitle title={`${template.data.title} | Templates | AgDS`} />
```

Comment:
```suggestion
			<DocumentTitle title={`${template.data.title} | Templates`} />
```

AgDS is appended in the `DocumentTitle`